code/index.html

```diff
@@ -2,29 +2,155 @@
+            <span class="remember"><em>Remember to claim your space.</em></span>
+
+            <h2 id="greeting"></h2>
+
+    </section>
+
+
+
+<div class="forms-container">
+    <form id=form action="https://httpbin.org/anything">
+
+      <span class="sign-up">sign up!</span>
+      
+        <label>First name:<br>
+          <input required type="text" id="firstName" for="firstname" name="name">
+        </label>
+
+        <label> Surname:<br>
+          <input type="text" for="surname" name="surname">
+        </label>
 
+        <label>Date of Birth:<br>
+              <input type="date" for="dob" name="dob" min="2006-01-01">
+        </label>
+
+        <label> E-mail:<br>
+                <input type="email" for="email" name="email">
+        </label>
+
+        <label id="gender">Gender:<br>
+          <select id="gender">
+                <option>Non-binary</option>
+                <option>Girl</option>
+                <option>Boy</option>
+                <option>Don't care</option>
+                </select>
+        </label>
+
+        <label for="parentsagreed">Parent's have agreed
+           <input class="checkbox-size" type=checkbox>
+        </label>
+
+        <label for="termsconditions">I agree to the terms and conditions
+           <input type=checkbox>
+        </label>
+
+        <button type="submit">Sign me up!</button>
+
+      
+    </form>
     <!-- Signup form -->
+    
 
+    
     <section class="accordion">
       <!-- Accordion -->
-      <h1>FAQ</h1>
+        <span class="sign-up say-what">say what!?</span>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+          <h1>FAQ</h1>
+
+          <div class="answer-open">
+            <h3 id="button"> What is this?</h3>
+              <p>This is a free community for the new generations to get to know more about the place we live in</p>
+         </div>
+
+         <div class="answer-open">
+           <h3 id="button">Does it cost?</h3>
+              <p>Not a single penny. We want you to be able to find fun and interesting facts in one place and at the same time meet friends who share your interest for the planet and the universe we live in.</p>
+         </div>
+         
+         <div class="answer-open">
+           <h3 id="button">How old do you need to be?</h3>
+              <p>Our target group is from born - 15 years old</p>
+         </div>
+
+         <div class="answer-open">
+           <h3 id="button">Why do my parents have to agree?</h3>
+              <p>Because they care about you.</p>
+         </div>
+
+         <div class="answer-open">
+           <h3 id="button">Can I cancel at anytime?</h3>
+              <p>Yep, of course! Just click link "unsubscribe" that comes with every newsletter.</p>
+         </div>
+
+      </section>
+    
+    </div>
+
```

Comment:
Do you remember about the `<main>` semantic element? Which defines the main section of a page which in your project seems to be from line 47 - 141.